ansible/configs/kni-osp/files/clouds.yaml.j2

```diff
@@ -0,0 +1,12 @@
+clouds:
+  {{ osp_project_name }}:
+    auth:
+      auth_url: "{{ osp_auth_url }}"
+      username: "{{ osp_auth_username }}"
```

Comment:
I think this needs to be changed to `{{ guid }}-user`. While we use the full admin account to create some resources (i.e. project, user, etc), the project user should be used from that point forward and the admin credentials should never be written to the bastion - or at least not kept there if they must be used temporarily.

Reply:
this is how I found it in the tree I copied from, trying those changes